lang/en.json

```diff
@@ -44,33 +44,13 @@
   "dfreds-convenient-effects.spellGiftOfAlacrity": "Gift of Alacrity",
   "dfreds-convenient-effects.spellGuidance": "Guidance",
   "dfreds-convenient-effects.spellHaste": "Haste",
-  "dfreds-convenient-effects.spellHuntersMark1h": "Hunter's Mark",
-  "dfreds-convenient-effects.spellHuntersMark8h": "Hunter's Mark (8h)",
-  "dfreds-convenient-effects.spellHuntersMark24h": "Hunter's Mark (24h)",
+  "dfreds-convenient-effects.spellHuntersMark": "Hunter's Mark",
   "dfreds-convenient-effects.spellLongstrider": "Longstrider",
   "dfreds-convenient-effects.spellMageArmor": "Mage Armor",
   "dfreds-convenient-effects.spellPassWithoutTrace": "Pass without Trace",
   "dfreds-convenient-effects.spellReduce": "Reduce",
   "dfreds-convenient-effects.spellShield": "Shield",
   "dfreds-convenient-effects.spellShieldOfFaith": "Shield of Faith",
-  "dfreds-convenient-effects.spellSlow": "Slow",
+  "dfreds-convenient-effects.spellSlow": "Slow"
 
-  "dfreds-convenient-effects.visionDim0": "Self",
-  "dfreds-convenient-effects.visionDim30": "Darkvision (30 ft)",
-  "dfreds-convenient-effects.visionDim60": "Darkvision (60 ft)",
-  "dfreds-convenient-effects.visionDim90": "Darkvision (90 ft)",
-  "dfreds-convenient-effects.visionDim120": "Darkvision (120 ft)",
-  "dfreds-convenient-effects.visionDim150": "Darkvision (150 ft)",
-  "dfreds-convenient-effects.visionDim180": "Darkvision (180 ft)",
-  "dfreds-convenient-effects.visionBright120": "Devil's Sight (Warlock)",
-
-  "dfreds-convenient-effects.lightNone": "None",
-  "dfreds-convenient-effects.lightCandle": "Candle",
-  "dfreds-convenient-effects.lightLamp": "Lamp",
-  "dfreds-convenient-effects.lightBullseye": "Lantern (Bullseye)",
```

Comment:
I think you probably still want the Lanterns and Torch, but just defined as they are around here: https://github.com/DFreds/dfreds-convenient-effects/blob/a478b94b5a0ee2b8e9f0661e9de5feb2b04ae43b/scripts/effects/effect-definitions.js#L2441

Reply:
Yes you are right, but for now let's leave the list of i18n keys alone, I will take it back for good when we have defined the management class